jobs.py

```diff
@@ -416,6 +416,15 @@
         "params": {},
         "paths": ["tests/fs_diff"],
     },
+    "systemd": {
+        "description": "Testing systemd failed units and dependency cycles",
+        "requirements": [
+            "A XCP-ng host >= 8.2"
+        ],
+        "nb_pools": 1,
+        "params": {},
+        "paths": ["tests/systemd"],
+    },
```

Comment:
I'd rather have it in the main job, not separated into its own job. Also, I'm not sure we need a new `systemd` directory in `tests`. `tests/system` could be a fit.

Reply:
Works for me. Will move it into system (it was there in the first place actually :))

Reply:
Fixed